README.md

```diff
@@ -136,6 +136,11 @@ By default, we do not show days from the previous month and the next month in th
    initialVisibleMonth: PropTypes.func,
 ```
 
+Optionally you can provide calendar caption block renderer.
+```js
+   renderCalendarCaption: PropTypes.func,
```

Comment:
Oh this is totes my fault by having the `Calendar props` story isn't it. What if we called this `renderCalendarAdditionalInfo` or something? Maybe even just `renderAdditionalInfo`? God, I'm verbose.